opencti-platform/opencti-graphql/src/utils/format.js

```diff
@@ -95,7 +95,7 @@ export const observableValue = (stixCyberObservable) => {
     case ENTITY_PROCESS:
       return stixCyberObservable.pid || stixCyberObservable.command_line || 'Unknown';
     case ENTITY_SOFTWARE:
-      return stixCyberObservable.name || 'Unknown';
+      return stixCyberObservable.name || stixCyberObservable.cpe || stixCyberObservable.swid || 'Unknown';
```

Comment:
You missed this TODO

// TODO for now this list is duplicated in Front, think about updating it aswell

You need to update the Entity.ts file in front or find a way to retrieve the observableValue from the back-end

Reply:
thanks @RomuDeuxfois , i've missed that 🙏